### PR TITLE
HW Lesson4

### PR DESCRIPTION
**Pull request by serghei-rubailo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,16 @@
-# saltedge-courses2020
```
Comment (jeneamocan) [inline README.md]: You've done a great job! There's just one thing you should do - discard the changes on the README file

**Review by jeneamocan — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (serghei-rubailo) [reply README.md]: Ok. Done.
README.md was discarded to "master" state.

**Review by jeneamocan — APPROVED** [review]

**Comment by jeneamocan** [comment]
Everything's great, just add the file into a folder, same as here https://github.com/mtkachenk0/saltedge-courses2020/pull/39

**Comment by serghei-rubailo** [comment]
> Everything's great, just add the file into a folder, same as here #39

Ok. Done.
"rubailo_s" directory was created. lesson4 HW file was moved to new directory.

**Pull request merged**